lib/serverless.js

```diff
@@ -183,6 +183,30 @@ function registerRootProxy (
     }))
 }
 
+function registerVpc (
+  target /* : string */,
+  vpcSecurityGroups /* : string */,
+  vpcSubnets /* : string */,
+  separator /* : string */
```

Comment:
Might not be necessary to have `separator` be a configurable thing. Instead of `.split()`ing and `.trim()`ing, what if we `.match()`ed a regular expression for what valid identifiers are supposed to look like?

Otherwise, what about just hardcoding the comma-separated approach so we don't need the 4th parameter?